### PR TITLE
getEventTarget: Fix getting element from nullish path

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

**Review by Zedwag — DISMISSED** [review]

File: packages/devextreme/js/events/utils/event_target.js [hunk 1]
```diff
@@ -6,12 +6,13 @@ export const getEventTarget = (event) => {
...
 
     const isShadowDOMUsed = Boolean(originalEvent.target?.shadowRoot);
+
     if(!isShadowDOMUsed) {
         return originalEvent.target;
     }
 
     const path = originalEvent.path ?? originalEvent.composedPath?.();
-    const target = path[0];
+    const target = path?.[0];
```
Comment (ksercs) [inline packages/devextreme/js/events/utils/event_target.js]: Are you sure it's okay to return undefined in this case?
I think it's more safe to return `event.target` (do nothing like we are not in shadowDOM)

Comment (marker-dao) [reply packages/devextreme/js/events/utils/event_target.js]: Yes, but it's incorrect behavior too: if target is incorrect, we get error which I fixed here: [https://github.com/DevExpress/DevExtreme/pull/27189]. Just a note. I will replace undefined with event.target.

Comment (ksercs) [reply packages/devextreme/js/events/utils/event_target.js]: As far as i understand in that case path always exists, not?

**Review by ksercs — APPROVED** [review]

File: packages/devextreme/js/events/utils/event_target.js (hunk 1 above)
Comment (marker-dao) [reply packages/devextreme/js/events/utils/event_target.js]: [71ea9871bd24d47524de443f41651d59038bc0e1]

**Review by Zedwag — APPROVED** [review]

**Pull request merged**